packages/webview_flutter/webview_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.2.0
+
+* Add ability to enable/disable zoom functionality
```

Comment:
nit: missing period at the end of the sentence.

Reply:
Fixed in cc9f536